templates/web/common/site_layout.tt.html

```diff
@@ -21,11 +21,11 @@
     <link rel="stylesheet" href="[% static_subdomain %]/css/dist/select2.min.css">
     <link rel="search" href="[% formatted_subdomain %]/cgi/opensearch.pl" type="application/opensearchdescription+xml" title="[% lang('site_name') %]">
 	[% header %]
-    <style media="all">
+    <!-- <style media="all">
         [% lang("css") %]
         [% styles %]
-    </style>
-    [% google_analytics %]
+    </style> -->
+    /* [% google_analytics %] */
```

Comment:
![image](https://user-images.githubusercontent.com/86596711/224494940-4b91475c-aa35-4155-86f0-1bd56d865c4a.png)

I commented the <style media='all'> tag because [% lang("css") %] was showing error. Which might be safely ignorable as you said.

Reply:
Quoting my self on slack (https://openfoodfacts.slack.com/archives/C01USC55N67/p1678551212810859)
> So, a lot of times IDEs/code editors don't recognise valid syntax if they don't have support for it (eg: for the templates ie, "*.tt.html") and throw errors/warnings for perfectly valid stuff.

What you're trying to fix is one of _**those**_ errors. You've effectively commented some code out and shuffled up the code a bit to remove IDE/code-editor errors. However, that syntax was perfectly valid for the templating engine that is used for the dynamic generation of html pages.